### ISSUE TITLE
[Suggestion] Rocket League colours

### ISSUE THREAD

**Issue by Joshiraptor** [SpoinkyNL/Artemis#64]
Was just talking about this on reddit, was told to suggest it here: Best effect for the boost lighting might be using the numbers bar in an orange colour, for a proper boost bar effect that players can see over their hand when using WASD. Also if only using numbers 1-10 it's in easy to read 10% increments.

Edit: Also when boosting have the colour change to yellow?

**Comment by JewsOfHazard** [comment]
There's currently two modes. Both of them work on the entire keyboard.

1. Spoinky Mode: The boost bar is a gradient from left to right

2. Hazard Mode: The boat bar is green when over 66% yellow when between 66 and 33 and red below 33. This also is the entire keyboard. 

Making it smaller wouldn't help and if you need to use an exact number I think you should look at the meter in the bottom left.

**Comment by SpoinkyNL** [comment]
Hey, setting it up like this will be very easy once the new configuration system is in place. Keep an eye on the GitHub page for more info on that once I have a proof of concept :)

**Comment by SpoinkyNL** [comment]
This is included in the latest update. Should you have any problems with it please create a new issue :)
